docs/webpack-dev-server.config.js

```diff
@@ -39,6 +39,8 @@ const config = {
   plugins: [
     // Allows for sync with browser while developing (like BrowserSync)
     new webpack.HotModuleReplacementPlugin(),
+    // Prevent moment from loading all the locales
+    new webpack.IgnorePlugin(/^\.\/locale$/, /moment$/),
```

Comment:
Sorry, lost my follow-up comment submitted on mobile, which basically said the same as your first paragraph. :+1: